tests/e2e-leg-6/verify-vrpq-conditions/30-create-vrpq-with-start-timestamp-filter.yaml

```diff
@@ -0,0 +1,20 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
+      export STARTTIMESTAMP=$(kubectl get vrpq archive-query -o json -n $NAMESPACE | jq -r '.status.restorePoints[] | select(.index==1 and .archive=="db") | .timestamp')
+      export ENDTIMESTAMP=$STARTTIMESTAMP
```

Comment:
Quick question: Do we need to define ENDTIMESTAMP since vrpq-start-timestamp.yaml only requires 'STARTTIMESTAMP'?

Reply:
same question for step 40